src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -19,6 +19,8 @@
 
 public final class Items {
 
+    static final Enchantment glowEnchant = Enchantment.getByKey(Constants.GLOW_ENCHANT);
```

Comment:
What is this supposed to be? A constant? If so it should be public and follow naming conventions. If just a global variable then it should be private.